.github/pull_request_template.md

```diff
@@ -1,6 +1,24 @@
-<body>
+<!--- Provide a summary of your changes in the title field above. -->
 
-<footer>
- 
+## Description
+<!--- Describe your changes. -->
+
+## Motivation & context
+<!--- What problem does this change solve? -->
+<!--- Provide a link if you are addressing an open issue. -->
+
+## Issue type checklist
+<!--- What type of change are you submitting? Check the boxes that apply: -->
+- [ ] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
+- [ ] **New feature**: non-breaking change that adds functionality.
+- [ ] **Breaking change**: fix or feature that will cause current functionality to change.
```

Comment:
So, we have labels for these, do we want to reserve labels for issues and use this instead? I'm fine with either.